translate/storage/lisa.py

```diff
@@ -343,7 +342,7 @@ def parse(self, xml):
             xml = posrc
         parser = etree.XMLParser(strip_cdata=False)
         self.document = etree.fromstring(xml, parser).getroottree()
-        self._encoding = self.document.docinfo.encoding
+        self.encoding = self.document.docinfo.encoding
```

Comment:
It seems that in this file we are extracting the encoding but not using it. Not an issue that needs to be addressed in this PR.